zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -203,19 +203,24 @@ mod tests {
         let re = Regex::new(r"\s+").unwrap();
 
         let expected_tables = vec![
-            "CREATE TABLE \"accounts\" (
-                account INTEGER PRIMARY KEY,
-                ufvk TEXT NOT NULL,
+            r#"CREATE TABLE "accounts" (
```

Comment:
I hadn't been aware of the`r#` prefix for raw string literals, this is good to know about.